### PR TITLE
Add a footer matching the current theme of the website

### LINKED ISSUE #14: Add Contact us page
#### Current Behavior
There is no contact page
#### Desired Behavior
Add `contact us` page with relevant social media and GitHub repo links, you can also automate and send out an email if a person clicks on any of the links

### PR DESCRIPTION
**Pull request by ampsteric**
**Description**
Hey! , I have added a footer that blends very well with the current theme of the website and it is also totally responsive!
Do check it out and let me.
This PR fixes #
This PR fixes #14 
**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by san-coding** [comment]
Hey, thanks you for contributing 🎉, could you share a screenshot of your changes

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (sudhay23) [inline .vscode/settings.json:1]: Can you make a `.gitignore` file that ignores this from being tracked

File: index.html [hunk 2]
```diff
@@ -156,11 +161,78 @@ <h2 class="text-gray-900 title-font text-lg font-medium">Fogg Wrist Watch</h2>
...
               </div>
+            </div> -->
+            <div class="footer">
+              <div class="heading">
+                <a class="flex title-font font-medium items-center text-gray-900 mb-4 md:mb-0">
+                  <img class="ml-3 text-xl" src="https://cdn3.iconfinder.com/data/icons/small/512/buy_cart_ecommerce_shopping_online_store-512.png" style="height: 50px;"></img>
+                  <span class="ml-3 text-xl"><text  style="font-size: 20px; font-family:'Montserrat', sans-serif;color: #000000;">ShopOnline</text></span>  
+              </a>
+              </div>
+              <div class="content">
```
Comment (sudhay23) [inline index.html:177]: Good that you've added a lot. But unfortunately these aren't the services the platform offers. Maybe you can modify content on page that aligns with the services the platform offers

**Review by sudhay23 — CHANGES_REQUESTED** [review]
@san-coding Can you continue the review? @ampsteric Good work btw :)

File: index.html (hunk 2 above)
Comment (ampsteric) [reply index.html:177]: okay sure , but can i get a list of all the links that you want to have in the footer , so that I don't miss out anything.

Comment (san-coding) [reply index.html:177]: you can add GitHub repo link, my twitter id @Sandeeprajakris and sudhay2001@gmail.com

**Comment by san-coding** [comment]
@ampsteric please share screenshot of whatever changes you have made , good job👍🏻

**Comment by ampsteric** [comment]
sure , adding screenshots of the footer below:
footer1
footer2

The footer is responsive and fits every view as per the orientation of the current website.

**Review by sudhay23 — APPROVED** [review]
Good job @ampsteric

**Review by san-coding — APPROVED** [review]
Good job

**Comment by ampsteric** [comment]
Thanks a lot , grateful that you liked it !

**Pull request merged**